lighthouse-cli/test/smokehouse/test-definitions/forms/form-expectations.js

```diff
@@ -284,6 +284,83 @@ const expectations = [
       requestedUrl: 'http://localhost:10200/form.html',
       finalUrl: 'http://localhost:10200/form.html',
       audits: {
+        'autocomplete': {
+          score: 0,
+          details: {
+            items: [
+              {
+                failingElements: {
```

Comment:
In lieu of unit tests (which I think the complexity of this audit jusssssst barely doesn't meet the threshold of), can you manually verify that all the autocomplete issues on the form elements page are correctly tracked in this smoke test expectation? Anything missing, or anything wrongly included here?

Reply:
> In lieu of unit tests (which I think the complexity of this audit jusssssst barely doesn't meet the threshold of), can you manually verify that all the autocomplete issues on the form elements page are correctly tracked

waiting until the audit implementation settles in this PR might be a good idea, but it seems like this is the perfect use of unit tests?